config/initializers/time.rb

```diff
@@ -1,7 +1,7 @@
 # Extends time class with convenience methods
 class Time
   def display_date
-    getlocal.strftime("%Y-%m-%d")
+    getlocal.strftime("%m-%d-%Y")
```

Comment:
Please revert this if you get a chance -- I'd like the standard date format to stay YYYY-MM-DD per iso standard. Kinda picky, I know, but I sometimes feel like my day job is 80% getting bitten in the behind by timezone and date format problems and 20% programming, haha.